engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.controller.js

```diff
@@ -0,0 +1,167 @@
+                }
+                $scope.syncState.set(stateFromTask($scope.syncTask));
+            });
+        }
+
+        function taskUpdated(task) {
+            if ($scope.syncTask.result !== undefined && $scope.syncTask.result !== 'pending'){
+                refreshSyncStatus();
+            }
+        }
+
+        function aggregateTasks(tasks) {
+            var taskIds = _.map(tasks, function (task) {
+                                return task.id;
+                            });
+            return AggregateTask.new(taskIds, taskUpdated);
+        }
+
+        function pickLastTask(tasks) {
+            var task = tasks.slice(-1)[0];
+            task.progressbar = {
+                value: task.progress * 100,
+                type: $scope.progressbarType(task)
+            }
+            return task;
+        }
+
+        function isTaskInProgress(task) {
+            return (task && (task.state === 'pending' || task.state === 'running'))
+        };
+
+        function stateFromTask(syncTask) {
+            if (isTaskInProgress(syncTask))
+                return syncState.SYNCING;
+            if (syncTask && syncTask.result !== 'success')
+                return syncState.FAILURE;
+            else
+                return syncState.DEFAULT;
+        };
+
+
+        $scope.syncState = syncState;
+
+        $scope.expandEnvironments = {};
+
+        refreshSyncStatus();
+
+        $scope.$on('$destroy', function () {
+            if ($scope.syncTask) {
+                $scope.syncTask.unregisterAll();
+            }
+        });
+
+        $scope.isTaskInProgress = isTaskInProgress;
+
+        $scope.syncCapsule = function () {
+            if (!$scope.syncState.is(syncState.SYNCING)) {
+
+                $scope.syncErrorMessages = [];
+                $scope.syncState.set(syncState.SYNC_TRIGGERED);
+
+                CapsuleContent.sync({id: capsuleId}, function (task) {
+                    $scope.syncStatus.active_sync_tasks.push(task);
+                    $scope.syncTask = aggregateTasks($scope.syncStatus.active_sync_tasks);
+                    $scope.syncState.set(syncState.SYNCING);
+                }, function (response) {
+                    $scope.syncErrorMessages = [response.data.displayMessage];
+                    $scope.syncState.set(syncState.DEFAULT);
+                });
+            }
+        };
+
+        $scope.cancelSync = function () {
+            if ($scope.syncState.is(syncState.SYNCING)) {
+
+                $scope.syncState.set(syncState.CANCEL_TRIGGERED);
+                CapsuleContent.cancel_sync({id: capsuleId}, function (response) {
+                }, function (response) {
+                    $scope.syncErrorMessages = [response.data.displayMessage];
+                });
+            }
+        };
+
+        $scope.syncStatusText = function (syncState, syncStatus) {
+            if (syncStatus === undefined)
+                return;
```

Comment:
Please use brackets for if statements. As well, you can use `angular.isUndefined(syncStatus)`